src/main/java/core/basesyntax/generator/ColorSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.generator;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int NUMBER_OF_COLORS = 5;
+
+    public String setColors() {
+        Random random = new Random();
+        String color;
+        int colorNumber = random.nextInt(NUMBER_OF_COLORS);
+        switch (colorNumber) {
+            case 0:
+                color = Colors.RED.toString();
+                break;
+            case 1:
+                color = Colors.GREEN.toString();
+                break;
+            case 2:
+                color = Colors.BLUE.toString();
+                break;
+            case 3:
+                color = Colors.YELLOW.toString();
+                break;
+            case 4:
+                color = Colors.PURPLE.toString();
+                break;
+            default:
+                color = "Haven`t color";
+                break;
+        }
+        return color;
```

Comment:
```suggestion
        int colorNumber = random.nextInt(NUMBER_OF_COLORS);
        switch (colorNumber) {
            case 0:
                return Colors.RED.toString();
            case 1:
                return Colors.GREEN.toString();
            case 2:
                return Colors.BLUE.toString();
            case 3:
                return Colors.YELLOW.toString();
            default:
                return Colors.PURPLE.toString();
        }
```
you can simplify this even more - have a look at `Colors.values()` method